examples/sparse_regression.py

```diff
@@ -0,0 +1,318 @@
+import argparse
+import functools
+
+import numpy as np
+import torch
+import math
+
+import pyro
+import pyro.distributions as dist
+from pyro import poutine
+from pyro import sample, param
```

Comment:
nit: i know this is ported from numpyro but it'd be nice to use `pyro.sample()` as in all the other tutorials

Reply:
ok